lib/jobs/wsman-job.js

```diff
@@ -51,169 +51,156 @@ function wsmanJobFactory(
+                    })
+                })
+                .catch(function (error) {
+                    logger.error("error occured " + error);
+                });
+            });
         });
-    }       
-    
-    
-	/*
-	*   Print the result for RestAPI Response
-	*/
-	
+    }
+
+
+    /*
+    *   Print the result for RestAPI Response
+    */
+
     WsmanJob.prototype.printResult = function (result) {
-		
-		logger.info(JSON.stringify(result,null,4));
-
-	};
-
-	
-	
-	WsmanJob.prototype.getPowerMonitoring = function() {
-		var self = this;
-				
-		var apiHost=self.apiServer;
-		var path=self.powerThermalConfigs.endpoints.powerthermal;
-		var method='POST';
-			
-		if (!self.userConfig){
-			throw ("No user configuration data provided ");
-		}
-		var data= {
-			
-				"serverAddress": self.oobServerAddress,
-				"userName" :self.userConfig.user,
-				"password" :self.userConfig.password,
-			};
-		
-		return self.clientRequest(apiHost,path,method,data);
-	} ;
-	
-	
-    
-    
+        logger.debug(JSON.stringify(result, null, 4));
+    };
+
+
+    WsmanJob.prototype.getPowerMonitoring = function () {
+        var self = this;
+
+        var apiHost = self.apiServer;
+        var path = self.powerThermalConfigs.powerthermal;
+        var method = 'POST';
+
+        if (!self.userConfig) {
+            throw ("No user configuration data provided ");
+        }
+        var data = {
+
+            "serverAddress": self.oobServerAddress,
+            "userName": self.userConfig.user,
+            "password": self.userConfig.password
+        };
+        return self.clientRequest(apiHost, path, method, data);
+    };
+
+
     /*
-	 * Client Request API
-	 * 
-	 * 
-	 */
-
-	WsmanJob.prototype.clientRequest = function(host, path, method, data) {
-		var self = this;
-
-		var parse = urlParse(host);
-
-		var setups = {};
-
-		setups.url = {};
-		setups.url.protocol = parse.protocol.replace(':','').trim();
-		setups.url.host = parse.host.split(':')[0];
-		setups.url.port = parse.port;
-		setups.url.path = path || '/';
-
-		setups.method = method || 'GET';
-		setups.credential = {};
-		setups.verifySSl = false;
-		setups.headers = {'Content-Type': 'application/json'};
-		setups.recvTimeoutMs = (30*1000);
-		setups.data = data || '';
-			
-		
-		self.printResult(setups);
-		
-		var http = new HttpTool();
-		return http.setupRequest(setups)
-		.then(function(){
-			return http.runRequest();
-		})
-		.then(function(response){
-		   if (response.httpStatusCode > 206) {
-			 var errorMsg = _.get(
-			 response,
-			 'body.error.message', 'IP is NOT valid or  httpStatusCode > 206');
-			throw new Error(errorMsg);
-		  }
-
-		if (response.body.length > 0) {
-			response.body = JSON.parse(response.body);
-		 }
-		return response.body;
-		});
-	};
-    
-    
-                  
-     return WsmanJob;
+     * Client Request API
+     * 
+     * 
+     */
+
+    WsmanJob.prototype.clientRequest = function (host, path, method, data) {
```

Comment:
How about removing this function and remove it with WsmanTool ? I know it was my bad to copy the function here